remix-tests/src/compiler.ts

```diff
@@ -21,6 +21,32 @@ function writeTestAccountsContract (accounts: string[]) {
     return testAccountContract.replace('>accounts<', body)
 }
 
+function processFile(filePath: string, sources: any, isRoot: boolean = false) {
```

Comment:
Add comments to describe what is the function doing.

Reply:
When I read `source` alongside `filePath` I expect `source` to be the text content of the file. But it seems not. Maybe find a better name. Also if this is a map, you can type it `Record<string, ....>`